src/pages/user/Settings.js

```diff
@@ -33,6 +44,14 @@ export default class UserSettings extends Component {
     this.setState({ filename: event.target.files[0].name });
   }
 
+  toggleDarkMode() {
```

Comment:
This logic should be in the new theme reducer - you should be able to call `this.props.actions.toggleDarkMode()` and be done with it, imo.

Reply:
Added toggleDarkMode() action and removed setTheme() action